### ISSUE TITLE
Data browser cannot open savedfile

### ISSUE THREAD

**Issue by MjBordalo** [ControlSystemStudio/cs-studio#2213]
Hi i am having a problem with BOY.
After i save a databrowser file and try to reopen it the working window gets grey (no databrowser layout).
Then if i close that windows i get this error:
An error has occurred. See error log for more details.
java.lang.NullPointerException

Help apreciated
Thanks in advance
MiguelB

**Comment by MjBordalo** [comment]
solved by unchecking:
`Use default  archives, ignoring *.plt file?`
under Edit >Preferences> CSS Applications>Trends>Data Browser

**Comment by kasemir** [comment]
Re-opening issue since I'll need to look into fixing the NullPointerException.
Good that you found a way to work around the error.

**Comment by kasemir** [comment]
Under Edit >Preferences> CSS Applications>Trends>Data Browser, do you have any default data sources configured, or is the list empty?

**Comment by MjBordalo** [comment]
Indeed. I have the same configurations as in Archive Data Servers URLs.
Was not supposed?

**Comment by kasemir** [comment]
Can you add a screenshot of your Data Browser settings, i.e. the ones that result in the null pointer error when you _check_ `Use default archives, ignoring *.plt file`?

**Comment by MjBordalo** [comment]
i believe it was like this.

![image](https://cloud.githubusercontent.com/assets/24882807/26794033/10c0a0a2-4a18-11e7-8397-60f6c2a7f7b9.png)
